cob_bringup/components/raw_mini_base.launch

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="sim" default="false"/>
+
+	<arg name="driver_yaml" value="$(arg pkg_hardware_config)/robots/common/raw_mini_base_driver.yaml"/>
+	<arg name="controller_yaml" value="$(arg pkg_hardware_config)/robots/common/raw_mini_base_controller.yaml"/>
+
+	<node pkg="rosserial_python" type="serial_node.py" name ="rosserial" output="screen">
+		<param name="port" value="/dev/ttyACM0"/>
+		<param name="baud" value="115200"/>
+
+		<remap from="/cmd_vel" to="/base/velocity_smoother/command"/>
+	</node>
+
+	<group ns="base">
+		<!-- twist mux -->
+		<include file="$(find cob_bringup)/tools/twist_mux.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="cmd_vel_out" default="velocity_smoother/command"/>
+			<arg name="marker_out" default="twist_mux/marker"/>
+		</include>
+
+		<!-- velocity smoother -->
+		<include file="$(find cob_bringup)/tools/velocity_smoother.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="config_file"           default="$(arg pkg_hardware_config)/robots/$(arg robot)/config/velocity_smoother.yaml"/>
+			<arg name="raw_cmd_vel_topic"     default="velocity_smoother/command"/>
+			<arg name="smooth_cmd_vel_topic"  default="twist_controller/command"/>
```

Comment:
indentation is inconsistent (here tabs, spaces in other files). unify to spaces since they are already in the majority of the files

Reply:
I'm sorry to say, but all launch files in `cob_bringup` use **TABS** for indentation...
please change accordingly